content/Hardware Support/Nano Family/Nano-boards-that-can-be-powered-directly-with-3-3-V.md

```diff
@@ -3,59 +3,80 @@ title: "Nano boards that can be powered directly with 3.3 V"
 id: 360014735580
 ---
 
-Some Nano boards have an exposed solder jumper that can be cut to enable powering the board directly with 3.3 V external power source.
+Some Nano boards have a trace that can be cut to bypass the step down converter, and enable powering the board directly with a 3.3 V external power source. This can provide greater power efficiency and reduced heat generation on the board, especially for applications when the board is running on battery power, and will be inactive for long periods of time.
```

Comment:
"Exposed trace" might be more accurate

Reply:
```suggestion
Some Nano boards have an exposed trace that can be cut to bypass the step down converter, and enable powering the board directly with a 3.3 V external power source. This can provide greater power efficiency and reduced heat generation on the board, especially for applications when the board is running on battery power, and will be inactive for long periods of time.
```